.travis/release.sh

```diff
@@ -22,6 +22,10 @@ sed "s/currentCSV: jaeger-operator.*/currentCSV: jaeger-operator.v${OPERATOR_VER
 sed "s~containerImage: docker.io/jaegertracing/jaeger-operator.*~containerImage: docker.io/${BUILD_IMAGE}~gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
 sed "s/name: jaeger-operator\.v.*/name: jaeger-operator.v${OPERATOR_VERSION}/gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
 sed "s~image: jaegertracing/jaeger-operator.*~image: ${BUILD_IMAGE}~gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
+
+export PREVIOUS_OPERATOR_VERSION=`grep "version: [0-9]" deploy/olm-catalog/jaeger-operator.csv.yaml | cut -f4 -d' '`
+sed "s/replaces: jaeger-operator\.v.*/replaces: jaeger-operator.v${PREVIOUS_OPERATOR_VERSION}/gi" -i deploy/olm-catalog/jaeger-operator.csv.yaml
```

Comment:
Shouldn't this be cumulative? Like, 1.11.0 replaces 1.10.0, 1.9.0, ... ?

Reply:
I think it is just a chaining mechanism - I've only looked at a couple of examples but they only listed the version they directly replaced.
@awgreene Is that correct?

Reply:
It is a chaining mechanism.